Sources/Apodini/Configurations/Configuration.swift

```diff
@@ -14,6 +14,16 @@ public protocol Configuration {
     /// - Parameter
     ///    - app: The `Vapor.Application` which is used to register the configuration in Apodini
     func configure(_ app: Application)
+    
+    /// A default CLI command that can be defined by the configuration.
+    /// This command is automatically integrated into the Apodini CLI,
+    /// if the `CommandConfiguration` has not been overridden.
+    var command: ParsableCommand.Type { get }
+    
+    // swiftlint:disable identifier_name
+    /// *For internal use only:* An array of the `command` of the configuration.
+    /// Used to allow iteration over the commands of a `ConfigurationBuilder`.
+    var _commands: [ParsableCommand.Type] { get }
```

Comment:
As far as I can see the comment has not been addressed yet. Is there a clear requirement to make this part of the protocol requirement. Also applies to my comment here: https://github.com/Apodini/Apodini/pull/326/files#r686172819

Reply:
@PSchmiedmayer As far as I tested, this needs to stay in the Configuration protocol  and public, otherwise the commands won't get added correctly to the cli, but just the default `EmptyCommand`

Reply:
@hendesi You are right, I have overlooked the dynamic calling behavior of protocol extensions. You are right that is is required to have that element in the protocol requirements. You can discard my comments about that 👍